portal-api/src/scheduler/jobs/helpers/utilisation-report-helpers.js

```diff
@@ -17,8 +17,12 @@ const DEFAULT_PAYMENT_OFFICER_TEAM_NAME = 'Team';
  */
 const getReportDueDate = async () => {
   const bankHolidays = await externalApi.bankHolidays.getBankHolidayDatesForRegion(BANK_HOLIDAY_REGION.ENGLAND_AND_WALES);
-  const businessDaysToAdd = process.env.UTILISATION_REPORT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH;
-  return addBusinessDaysWithHolidays(startOfMonth(new Date()), businessDaysToAdd, bankHolidays);
+  const businessDay = Number.parseInt(process.env.UTILISATION_REPORT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH, 10);
+
+  const reportDate = new Date();
+  const oneIndexedReportMonth = reportDate.getMonth() + 1;
+  const reportYear = reportDate.getFullYear();
+  return getBusinessDayOfMonth(oneIndexedReportMonth, reportYear, bankHolidays, businessDay);
```

Comment:
Just overheard your conversation, very least would be calling this getBusinessDayOfOneIndexedMonth, best case is probably what you just discussed with Mike of passing in only a date!